compiler/surface/lexer.cppo.ml

```diff
@@ -819,6 +819,15 @@ let line_dir_arg_re =
       eol
     ])
 
+let line_dir_arg_upcase_re =
```

Comment:
What does this catch?

Reply:
Yeah here we are using a low-cost parser on the directive lines, but the trouble is that e.g. the french translation uses many words (`> Usage de MyModule en tant que M`) so recovering the actual parameters can't be done with simple word indexing (the previous version would only return the last `M` here). Rather than enriching the line-lexer for sub-tokens on these directives and then a more serious parser, the cheaper solution here was to just rely on capitalisation of the module names :sweat_smile:

Reply:
Woupsy that's kind of sketchy, this deserves at least a comment :)